src/consumer/__tests__/runner.spec.js

```diff
@@ -40,17 +56,27 @@ describe('Consumer > Runner', () => {
       assigned: jest.fn(() => []),
       isLeader: jest.fn(() => true),
     }
-    const instrumentationEmitter = new InstrumentationEventEmitter()
+    instrumentationEmitter = new InstrumentationEventEmitter()
     runner = new Runner({
+      consumerGroup,
+      instrumentationEmitter,
+      logger: newLogger(),
+      eachBatch,
+    })
+    runnerPool = createRunnerPool({
```

Comment:
This looks a bit jacked up now. Shouldn't this test suite be testing the RunnerPool now rather than the Runner, since the runner is encapsulated within the RunnerPool? Do we need to instantiate the runner manually here or can we do all the testing via the runnerpool?